sermant-plugins/sermant-springboot-registry/springboot-registry-service/src/main/java/io/sermant/discovery/service/lb/discovery/zk/ZkClient.java

```diff
@@ -51,7 +51,11 @@ public ZkClient() {
 
     @Override
     public void start() {
-        if (!PluginConfigManager.getPluginConfig(DiscoveryPluginConfig.class).isEnableRegistry()) {
+        final String registryCenterType = PluginConfigManager.getPluginConfig(DiscoveryPluginConfig.class)
+                .getRegistryCenterType();
+        String zkName = "Zookeeper";
+        if (!PluginConfigManager.getPluginConfig(DiscoveryPluginConfig.class).isEnableRegistry()
```

Comment:
the result of PluginConfigManager.getPluginConfig(DiscoveryPluginConfig.class)  should save by temporary variable，only get once